.github/.wordlist.txt

```diff
@@ -1187,6 +1188,7 @@ riscv
 rloc
 rmw
 rodata
+rojopolis
```

Comment:
this does not look like a word. I believe we should escape (eg. via backticks) to not have spellcheck validate these as words.

Reply:
Removed.